app/src/main/res/layout/activity_ao_oneplus.xml

```diff
@@ -0,0 +1,105 @@
+            android:layout_marginTop="@dimen/aod_spacing_2"
+            android:layout_marginBottom="@dimen/aod_spacing_2"
+            android:capitalize="none"
+            android:fontFamily="@font/slate_medium"
+            android:gravity="center"
+            android:inputType="textCapWords"
+            android:text="@string/date"
+            android:textAllCaps="false"
+            android:textSize="18sp" />
+
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/aod_spacing"
+            android:layout_marginBottom="@dimen/aod_spacing"
+            android:layout_gravity="center"
+            android:orientation="horizontal"
+            tools:ignore="UseCompoundDrawables">
+
+            <TextView
+                android:id="@+id/batteryTxt"
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="1"
+                android:fontFamily="@font/slate_regular"
+                android:gravity="start"
+                android:text="@string/battery"
+                android:textSize="25sp" />
+
+            <ImageView
+                android:id="@+id/batteryIcn"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:contentDescription="@string/feature"
+                app:srcCompat="@drawable/ic_battery_unknown" />
+
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/aod_spacing"
+            android:layout_marginBottom="@dimen/aod_spacing"
+            android:layout_gravity="center"
+            android:orientation="horizontal"
+            tools:ignore="UseCompoundDrawables">
+
+            <androidx.recyclerview.widget.RecyclerView
+                android:id="@+id/notifications_grid"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:layout_marginTop="@dimen/aod_spacing"
+                android:layout_marginBottom="@dimen/aod_spacing"
+                tools:itemCount="3">
+
+            </androidx.recyclerview.widget.RecyclerView>
+
+            <TextView
+                android:id="@+id/notifications"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/aod_spacing"
+                android:layout_marginBottom="@dimen/aod_spacing"
+                android:fontFamily="@font/slate_regular"
+                android:gravity="center"
+                android:text="@string/feature"
+                android:textSize="25sp" />
+        </LinearLayout>
```

Comment:
Please look at the other layouts and adjust the layout because the notification count and the notification icons are two separate things.